ansible/roles/software/ckan/inventory/handlers/main.yml

```diff
@@ -3,8 +3,13 @@
   service: name=apache2 state=restarted
   when: datagov_in_service
 
-- name: reload supervisor
-  command: supervisorctl reload
+- name: reread supervisor
```

Comment:
Minor, but I would leave this as `reload` for consistency with other services. The fact that we're doing reread instead of reload is just a supervisor implementation detail detail. When tasks are notifying supervisor, all they're saying is "supervisor, you need to reload your configuration files", so "reload supervisor" is a better name IMHO.